health_vara.py

```diff
@@ -184,6 +184,17 @@ def __setup__(cls):
             setattr(patient_field, 'readonly', False)
             setattr(patient_field, 'required', get_party_val(patient_field_name, 'required', True))
 
+        # Buttons
+        cls._buttons.update({
+            'create_imaging_request': {},
+        })
+
+    @classmethod
+    @ModelView.button
+    @ModelView.button_action('health_imaging.patient_imaging_test_request')
+    def create_imaging_request(cls, records):
```

Comment:
The view definition seems to use `create_imaging_request` like any other field on a model. Just curious if it is possible to create view elements (buttons, links, etc.) as fields on a model?

Reply:
Not sure I completely understand, but I think this is not possible. `fields` doesn't have an option for button, and I think the reason is that the value of `fields` gets persisted, whereas there is nothing sensible to persist for buttons and links.